app/src/org/commcare/android/fragments/SetupKeepInstallFragment.java

```diff
@@ -0,0 +1,61 @@
+package org.commcare.android.fragments;
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import org.commcare.android.framework.UiElement;
+import org.commcare.android.view.SquareButtonWithText;
+import org.commcare.dalvik.R;
+import org.javarosa.core.services.locale.Localization;
+
+/**
+ * Created by dancluna on 4/10/15.
+ */
+public class SetupKeepInstallFragment extends Fragment {
+    @UiElement(R.id.btn_start_install)
+    SquareButtonWithText btnStartInstall;
+
+    @UiElement(R.id.btn_stop_install)
+    SquareButtonWithText btnStopInstall;
+
+    public interface StartStopInstallCommands {
+        void onStartInstallClicked();
+        void onStopInstallClicked();
+    }
+
+    StartStopInstallCommands buttonCommands;
+
+    public void setButtonCommands(final StartStopInstallCommands buttonCommands) {
+        this.buttonCommands = buttonCommands;
+    }
+
+    @Override
+    public View onCreateView(final LayoutInflater inflater, final ViewGroup container, final Bundle savedInstanceState) {
+        final View view = inflater.inflate(R.layout.setup_keepinstall, container, false);
+        btnStartInstall = (SquareButtonWithText) view.findViewById(R.id.btn_start_install);
+        btnStartInstall.setText(Localization.get("install.button.start"));
```

Comment:
@ctsims not sure but wouldn't declaring these buttons as UIElements with localization ids handle this for you?